ExilenceNextApp/src/components/notification-list/notification-list-item/NotificationListItem.tsx

```diff
@@ -30,7 +30,11 @@ const useStyles = makeStyles((theme: Theme) => ({
     }
   },
   timestamp: {
-    display: 'inline'
+    display: 'inline',
+    fontSize: '12px'
+  },
+  notificationItem: {
+    fontSize: '14px'
```

Comment:
I think we need to change margins/paddings as well, but not sure yet. I'll check this branch out later on to see how it looks!

Reply:
Scratch that, tested this locally and it looks great.